modules/weko-records/weko_records/api.py

```diff
@@ -1878,16 +1878,39 @@ def get_item_link_info_output_xml(cls, recid):
         :return ret: List destination records.
         """
         from weko_deposit.api import WekoRecord
+        from weko_records_ui.utils import get_record_permalink
+
+        def get_url(pid_value):
+            wr = WekoRecord.get_record_by_pid(pid_value)
+            permalink = get_record_permalink(wr)
+            if not permalink:
+                sid = 'system_identifier_doi'
```

Comment:
Hi,

Plz create config var for this.

Reply:
This part seems difficult to have a standard config